composer.json

```diff
@@ -18,7 +18,8 @@
     "require-dev": {
         "athletic/athletic": "~0.1",
         "phpunit/PHPUnit": "~4.0",
-        "squizlabs/php_codesniffer": "^2.6.2"
+        "squizlabs/php_codesniffer": "^2.6.2",
+        "zendframework/zend-coding-standard": "~1.0.0"
```

Comment:
We can remove the dependency on php_codesniffer, as it's included in the zend-coding-standard. :smile: